BioArchLinux/rstudio-desktop/lilac.yaml

```diff
@@ -0,0 +1,23 @@
+build_prefix: extra-x86_64
+maintainers:
+  - github: kbipinkumar
+    email: kbipinkumar@pm.me  
+pre_build_script: |
+  update_pkgver_and_pkgrel(_G.newver.lstrip('v'))
```

Comment:
`.lstrip('v')` is unnecessary, because the `prefix: v` option to nvchecker already removes the `v` prefix.